src/pl/plpython/plpy_subxactobject.c

```diff
@@ -93,6 +94,7 @@ PLy_subtransaction_new(PyObject *self, PyObject *unused)
 	ob->started = false;
 	ob->exited = false;
 
+	PLy_enter_python_intepreter = true;
```

Comment:
did you process PLy_enter_python_intepreter in PLy_output?

Reply:
Thanks, fixed